### PR TITLE
Add instructions for cloning the repository and creating a new branch

### PR DESCRIPTION
**Pull request by KQuiggins**
### Summary

This pull request adds a new section to the README.md file about cloning the repository and creating a new branch for feature work.

### Motivation

The motivation behind these changes is to provide clear instructions for contributors on how to clone the repository and create a new branch for their feature work.

### Additional Information

Please review and provide any necessary feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,33 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
+### Clone the Repository
+
+1. Open your terminal or command prompt.
+2. Navigate to the directory where you want to clone the repository.
+3. Run the following command to clone the repository:
+
+    ```bash
```
Comment (coltonehrman) [inline README.md:11]: I noticed some whitespace in front of this. Can we format it correctly and remove extra whitespace.

**Comment by coltonehrman** [comment]
@KQuiggins This is great! Thanks for putting this together. I think the only thing missing is the install step, `npm install`.

**Comment by KQuiggins** [comment]
I will work on this tomorrow and fix the white space issue and add the npm install command. Thanks for taking a look at it for me.

**Comment by KQuiggins** [comment]
I have updated the commit by removing the whitespace and also added directions to install dependencies upon cloning the project.

**Review by coltonehrman — APPROVED** [review]
🎉

**Pull request merged**